project_report/tex/design.tex

```diff
@@ -1 +1,16 @@
 \section{Design}
+
+\subsection{Dashboard design}
+A very important aspect the group has had to concider, is which design pattern
+to model the dashboard after. A design pattern is a form of template, which in
+our case, dictates how the different classes communicate. A relevant design
+pattern is the MVC (Model - View - Controller). This pattern separates the
+database (model), the logic (controller) and the GUI (View) into different
+layers, and thereby helps keep separation of concerns. The controller manages
+input from the user, and forwards it to either the model, to retrieve or store
+data, or the view, to present data to the user.
+
+The group has decided to use the MVC design pattern, thus making it possible to
+work on the different aspects of the system (GUI, database, logic) separately.
```

Comment:
"... different parts of the system ..." 

**Aspect**
     _Noun_: A way in which something can be viewed by the mind.
     _Noun_: A characteristic or feature of something.